modules/util/args/CalculateLossArgs.py

```diff
@@ -9,16 +9,20 @@ class CalculateLossArgs(BaseArgs):
     output_path: str
 
     def __init__(self, data: list[(str, Any, type, bool)]):
-        super(CalculateLossArgs, self).__init__(data)
+        super().__init__(data)
 
     @staticmethod
-    def parse_args() -> 'CalculateLossArgs':
+    def parse_args() -> "CalculateLossArgs":
         parser = argparse.ArgumentParser(description="One Trainer Loss Calculation Script.")
 
         # @formatter:off
```

Comment:
Again, the `# @formatter:off`/ `# @formatter:on` was used to disable the formatter for some code blocks. This behavior should be kept

Reply:
Disabled formatter.